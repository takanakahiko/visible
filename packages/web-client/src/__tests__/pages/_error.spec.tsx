import { cleanup } from '@testing-library/react';
import React from 'react';

import Error from '../../pages/_error';
import { render } from '../../tests/render';

describe('NotFound', () => {
  afterEach(() => {
    cleanup();
  });

  it('matches to the snapshot', () => {
    const { container } = render(
      <Error name="name" message="message" statusCode={503} stack="foo" />,
    );
    expect(container).toMatchSnapshot();
  });
});
