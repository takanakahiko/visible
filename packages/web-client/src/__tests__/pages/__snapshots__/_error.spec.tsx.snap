// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound matches to the snapshot 1`] = `
<div>
  <main
    class="sc-Axmtr LSOvs"
  >
    <h1
      class="sc-fznZeY jOdKoG"
    >
      503: 
      name
    </h1>
    <p
      class="sc-fznZeY Cnwrk"
    >
      message
    </p>
    <pre
      style="display: block; overflow-x: auto; padding: 0.5em; background: rgb(240, 240, 240); color: rgb(68, 68, 68);"
    >
      <code>
        <span
          style="font-weight: bold;"
        >
          foo
        </span>
      </code>
    </pre>
  </main>
</div>
`;
